rake newclear issues

Now that `bluepotion create` is working I tried `rake newclear`

and everything went along smoothly. All compiles were fine and it started creating until it crashed:

```
   Compile ./app/screens/home_screen.rb
    Create ./build/Development-16/lib/armeabi/libpayload.so
    Create ./build/Development-16/lib/armeabi/gdbserver
    Create ./build/Development-16/lib/armeabi/gdb.setup
./build/Development-16/java/com/your_company/myapp/Request.java:4: cannot find symbol
symbol: class Com_Android_Volley_Toolbox_StringRequest
public class Request extends Com_Android_Volley_Toolbox_StringRequest {
                             ^
warning: com/rubymotion/String.class(com/rubymotion:String.class): major version 51 is newer than 50, the highest major version supported by this compiler.
It is recommended that the compiler be upgraded.
./build/Development-16/java/com/your_company/myapp/Request.java:49: package com.android.volley does not exist
	    protected java.util.Map<java.lang.String,java.lang.String> getParams() throws com.android.volley.AuthFailureError {
	                                                                                                    ^
./build/Development-16/java/com/your_company/myapp/Request.java:54: package com.android.volley does not exist
	    public java.util.Map<java.lang.String,java.lang.String> getHeaders() throws com.android.volley.AuthFailureError {
	                                                                                                  ^
./build/Development-16/java/com/your_company/myapp/Request.java:48: method does not override or implement a method from a supertype
	    @Override
	    ^
./build/Development-16/java/com/your_company/myapp/Request.java:53: method does not override or implement a method from a supertype
	    @Override
	    ^
5 errors
1 warning
rake aborted!
Command failed with status (1): [/usr/bin/javac -d "./build/Development-16/...]
/Library/RubyMotion/lib/motion/project/template/android.rb:526:in `block (2 levels) in <top (required)>'
/Library/RubyMotion/lib/motion/project/paralel_builder.rb:47:in `call'
/Library/RubyMotion/lib/motion/project/paralel_builder.rb:47:in `block (2 levels) in initialize'
Tasks: TOP => build
(See full trace by running task with --trace)
```

I've been having that compiler upgrade for a while so I decided to try to fix it before reporting.
I was able to get rid of the compiler warning by upgrading to the latest jdk7 (I first did jdk8 but was overwhelmed by the warnings), but the error stands starting with this error:

```
./build/Development-16/java/com/your_company/myapp/Request.java:4: error: cannot find symbol
public class Request extends Com_Android_Volley_Toolbox_StringRequest {
                             ^
  symbol: class Com_Android_Volley_Toolbox_StringRequest
```

I'm happy to alpha test BluePotion, let me know if there's anything I can do :)

--- a/files/Gemfile
+++ b/files/Gemfile
@@ -1,7 +1,10 @@
 source "https://rubygems.org"
 
 gem "rake"
-gem "bluepotion", github: "infinitered/bluepotion" 
+gem "bluepotion", github: "infinitered/bluepotion"
+
+# Managing Java-based dependencies
+gem "motion-gradle"
 
 # Debugging tools
 gem "newclear"

--- a/files/Rakefile.erb
+++ b/files/Rakefile.erb
@@ -7,7 +7,7 @@ Bundler.require
 
 Motion::Project::App.setup do |app|
   # Use `rake config' to see complete project settings.
- 
+
   app.api_version = "16"
   app.target_api_version = "16"
 
@@ -24,5 +24,11 @@ Motion::Project::App.setup do |app|
   app.application_class = "BluePotionApplication"
   app.main_activity = "PMHomeActivity"
   app.sub_activities += %w(PMSingleFragmentActivity PMNavigationActivity)
+
+  app.gradle do
+    # Google's networking API for Android
+    dependency "com.mcxiaoke.volley", :artifact => "library", :version => "1.0.10"
+  end
+
 end
 

